docs/sql-migration-guide.md

```diff
@@ -23,8 +23,11 @@ license: |
 {:toc}
 
 ## Upgrading from Spark SQL 3.0 to 3.1
+
```

Comment:
nit: unneeded change?

Reply:
It seems the other parts (e.g., `## Upgrading from Spark SQL 2.4.4 to 2.4.5`) have a single blank line in the head. So, I added a blank here, too.

Reply:
A space between `##` and `-` is actually a required change for the correct MD syntax IIRC. Morden tools support no spaces between them, but technically it's required for strict MD syntax.

Reply:
Ah, I see.